tests/RME/gold/make_gold_stations

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Very minor and I usually follow:
```
#! /usr/bin/env bash
```

[Discussion](https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash)

Reply:
I like that. One of those T.I.L. moments...